src/Data/Functor/Variant.purs

```diff
@@ -97,37 +102,37 @@ instance traversableCons ::
   ( IsSymbol k
   , TF.Traversable f
   , TraversableVFRL rl r
-  , R.Cons k (FProxy f) r r'
+  , R.Cons k f r r'
   , R.Union r rx r'
-  ) => TraversableVFRL (RL.Cons k (FProxy f) rl) r' where
+  ) => TraversableVFRL (RL.Cons k f rl) r' where
   traverseVFRL _ f = on k (TF.traverse f >>> map (inj k))
-    (traverseVFRL (RLProxy :: RLProxy rl) f >>> map expand)
-    where k = SProxy :: SProxy k
+    (traverseVFRL (Proxy :: Proxy rl) f >>> map expand)
+    where k = Proxy :: Proxy k
 
 instance foldableVariantF ::
   (RL.RowToList row rl, FoldableVFRL rl row) =>
   TF.Foldable (VariantF row) where
-    foldr = foldrVFRL (RLProxy :: RLProxy rl)
-    foldl = foldlVFRL (RLProxy :: RLProxy rl)
-    foldMap = foldMapVFRL (RLProxy :: RLProxy rl)
+    foldr = foldrVFRL (Proxy :: Proxy rl)
+    foldl = foldlVFRL (Proxy :: Proxy rl)
+    foldMap = foldMapVFRL (Proxy :: Proxy rl)
 
 instance traversableVariantF ::
   (RL.RowToList row rl, TraversableVFRL rl row) =>
   TF.Traversable (VariantF row) where
-    traverse = traverseVFRL (RLProxy :: RLProxy rl)
+    traverse = traverseVFRL (Proxy :: Proxy rl)
     sequence = TF.sequenceDefault
 
 -- | Inject into the variant at a given label.
 -- | ```purescript
 -- | maybeAtFoo :: forall r. VariantF (foo :: FProxy Maybe | r) Int
```

Comment:
Should this `FProxy` be removed?

Reply:
This is being highlighted badly in the review -- I mean the comment `maybeAtFoo` on `inj`, which still refers to `FProxy` but doesn't need to, previously on line 122 and now on line 127.